lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -178,12 +201,69 @@ def _install_internal_conntrack_flow(self):
                                              reset_default_register=False,
                                              resubmit_table=self.config.internal_conntrack_fwd_tbl)
 
+    def _send_messsage_interface(self, ue_ip_add:str):
```

Comment:
```suggestion
    def _send_message_interface(self, ue_ip_add:str):
```

Reply:
Also please run py format on the committed python files

Reply:
Yes. Naming was changed.